src/main/java/dk/cngroup/lentils/repository/UserRepository.java

```diff
@@ -2,11 +2,19 @@
 
 import dk.cngroup.lentils.entity.User;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
 import java.util.Optional;
 
 @Repository
 public interface UserRepository extends JpaRepository<User, Long> {
     Optional<User> findByUsername(String username);
+
+    @Query(value = "SELECT ua.* FROM user_account ua " +
+            "LEFT JOIN user_role ur ON ua.user_id = ur.user_id " +
+            "WHERE ua.user_id IN ?1 AND ur.role_id = ?2",
+            nativeQuery = true)
+    List<User> findAllByIdAndRoleId(List<Long> ids, Long role);
```

Comment:
Tam je nejaka moznost napsat jmeno metody, aby ti to vyrobilo i ten join a vyhnul ses nativnimu dotazu.

Kdyby sis nebyl jisty, mrknu na to.

Reply:
Neco jako tohle by mohlo fungovat:
findAllByIdAndRole_roleId

Reply:
Jo sorry, jeste potrebujes to IN. Tak co treba tohle
findAllByIdInAndRole_roleId

Reply:
vyzkouším :+1: